### ISSUE TITLE
[BCES01086] Regression: LBP2 audio cuts shortly followed by freeze

### ISSUE THREAD

**Issue by B-Knight49** [RPCS3/rpcs3#8482]
## Quick summary
After some time in game (~1.5 hours), the game will have its audio cut out. About 10-20 seconds later, the game will freeze and will need to be forcefully quit. 

## Details
Since build 10522 (from what I can tell), LBP2 freezes after roughly 1-2 hours in-game. It seems random and no error code is thrown as it doesn't explicitly crash. Shortly before the crash, audio cuts out. 10-20 seconds later, the screen will freeze and you will need to forcefully quit the process. 

This freeze has happened to me multiple times. Loading levels, exiting levels to my pod, on my moon creating a level, playing levels in Story Mode, getting an object out of my inventory, anything. 

#### 1. Please provide _exact_ build (or commit) information that introduced the regression you're reporting.
Starting from build 10522 / #8454. I believe. Please let me know if anyone has encountered it elsewhere. 

#### 2. Please attach TWO logs:
LBP2 logs are huge. Links to MEGA.nz:
* **(v10522)** Regression: 250MB / 6GB / About 1 hour varied gameplay 
https://mega.nz/file/IrB0kITa#YPQ6LcTwOsFWJO0n-jHy1PqJh0B8r0aXYx_vILf6HZY
* **(v10486)** No Crash: 550MB / 14GB / About 2 hours varied gameplay
https://mega.nz/file/86oy3ArD#upqsX3OVTg4cSfL459USZdrGBvZMhz8Q6czT0h_xuU8

3-5 not applicable. Non-RSX regression. 

#### 6. Please provide your system configuration:
* Windows 10 Home v1909
* i9-9900K @ 5.0Ghz
* MSI GTX 980Ti 6G Gaming
* v446.14

**Comment by B-Knight49** [comment]
10522 is bugged.
[Log uploading]. 250MB

10500 was fine. 
No log. Was ~23GB. But didn't crash so should be the same as 10486.

Will try 10516 now.

**Comment by B-Knight49** [comment]
Ignore everything here. It seems like a recent driver update has started causing these problems for me just out of the blue. It's related to my headset driver. 

The freeze just happened on build 10466. A build which I used to previously complete the entire game in a single sitting. I'd spent about 7 hours in-game with no problems. It must have been just a massive coincidence that I tested builds without any issues and came across it once or twice in others.
